x-pack/platform/plugins/shared/onechat/public/application/context/send_message/send_message_context.tsx

```diff
@@ -18,11 +19,20 @@ interface SendMessageState {
   canCancel: boolean;
   cancel: () => void;
   cleanConversation: () => void;
+  connectorSelection: {
+    connectors: ReturnType<typeof useConnectorSelection>['connectors'];
+    selectedConnector: string | undefined;
+    selectConnector: (connectorId: string) => void;
+    isLoading: boolean;
+    defaultConnectorId?: string;
+  };
```

Comment:
NIT: use `UseConnectorSelectionResult` here (or at least don't use `ReturnType<type of...>` in non-test code.

Reply:
code removed